### DISCUSSION TITLE
Spades installation

### DISCUSSION THREAD

**Discussion by DenisGoryunov** [ablab/spades#1046] [Q&A] [answered]
Hi,

I'm wondering if `conda` (https://anaconda.org/bioconda/spades) is a recommended way of Spades installation? There are not any mentions of `conda` in the `Spades` installation chapter of the documentation. 
I found out an old discussion here (https://github.com/ablab/spades/issues/30) but not shure if it's still relevant.
Could you help me please?
Regards,
Denis

**Comment by asl** [comment, marked as answer]
conda build of SPAdes is maintained by the community. The only supported way is via pre-built binaries and / or compiling from sources.

**Reply by DenisGoryunov** [reply]
Thanks a lot for your prompt reply!
